test/integration/test-example-validation.js

```diff
@@ -67,6 +68,9 @@ describe('example', function() {
 
     // TODO(dvoytenko): Remove once validator supports "amp-font" element.
     /DISALLOWED_TAG amp-font/,
+
+    // Remove once validator supports "data-account" for "amp-brightcove".
+    /MANDATORY_ATTR_MISSING account/
```

Comment:
I believe this should be instead:
```
/DISALLOWED_TAG amp-brightcove/,
```

Data-attributes are all whitelisted by default. But our validator doesn't know anything about "amp-brightcove" element yet - hence whitelist.